src/svc.h

```diff
@@ -332,7 +332,10 @@ static inline char *svc_status(svc_t *svc)
 		return "unknown";
 
 	case SVC_DONE_STATE:
-		return "done";
+		if (svc->started)
+			return "succeeded";
+		else
+			return "failed";
```

Comment:
I was tempted to merge this as-is, but I'm afraid I cannot approve this particular part, because there are tools out there (e.g. at $DAYJOB) that parse this output, expecting the string "done", and would break with this change.

If you change back to "done", keeping "failed", and restore the formatting above, then I can merge it.

Reply:
@troglobit sure, I can do that.